Snacks/Utilities/WindowUtils.cs

```diff
@@ -38,7 +38,7 @@ public static class WindowUtils
     {
         const double SECONDS_PER_MINUTE = 60.0;
```

Comment:
```suggestion
        double SECONDS_PER_MINUTE 
        {
            get
            {
                return KSPUtil.dateTimeFormatter.Minute
            }
        }
```
Property in case the value changes during play

Reply:
I think I would have liked to have done something similar here, but didn't know the correct C# to do it with. A direct assignment is not valid? ~~I would have thought that the parent class is already active~~, I guess there isn't a parent class yet here.

I think in looking at kronometer that all of the time intervals could be customized, minutes included, but I have no idea if anyone has implemented that in a planet pack yet. But, I am in agreement with you that it is good to handle as many cases by variable as possible.